.github/scripts/update_schutzfile.py

```diff
@@ -0,0 +1,35 @@
+#!/bin/python3
+
+import json
+import os
+import re
+
+REPO_FILES=os.listdir("/repo/")
+SINGLETONS=[]
+# Get a list of all repositories that contain 'singleton'
+for REPO_FILE in REPO_FILES:
+    with open("/repo/"+REPO_FILE, "r") as f:
+        data = json.load(f)
+    if ( "singleton" in data.keys() ):
+        SINGLETONS.append(data['snapshot-id'])
+
+REPO=os.environ.get('REPO')
+with open(REPO+"/Schutzfile", "r") as f:
+    data = json.load(f)
+
+URLS=[]
+ARCHES=['x86_64', 'aarch64']
+SUFFIX=os.environ.get('SUFFIX')
+# Replace snapshot SUFFIX in repositories that don't have 'singleton' present
+for key in data.keys():
+    if ("repos" in data[key].keys()):
+        for i in range(len(data[key]['repos'])):
```

Comment:
I think we can even do `for repo in data[key]['repos']` here.

Reply:
Actually not cause I need numbers to index the list in the dict. Maybe if there's a simpler way to get the length?

Reply:
`for key, value in enumerate(data[key]['repos'])`

Reply:
> `for key, value in enumerate(data[key]['repos'])`

Perfect, thanks!